gensim/corpora/bleicorpus.py

```diff
@@ -35,14 +32,25 @@ class BleiCorpus(IndexedCorpus):
 
     The vocabulary is a file with words, one word per line; word at line K has an
     implicit ``id=K``.
+
     """
 
     def __init__(self, fname, fname_vocab=None):
         """
         Initialize the corpus from a file.
 
-        `fname_vocab` is the file with vocabulary; if not specified, it defaults to
-        `fname.vocab`.
+        Parameters
+        ----------
+        fname : str
+            Serialized corpus's filename
+        fname_vocab : str or None, optional
```

Comment:
Need to understand how to:
- Document multiple types of argument (i.e. when the parameter can be type `X` or `Y`)
- Document multiple types for "Return" section
- Correctly specify the parent class (if there are many heirs)